packages/components/src/custom-select/index.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import Downshift, { useSelect } from 'downshift';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Dashicon } from '../';
+
+const itemToString = ( item ) => item.name;
+// This is needed so that in Windows, where
+// the menu does not necessarily open on
+// key up/down, you can still switch between
+// options with the menu closed.
+const stateReducer = (
```

Comment:
related to the Dragon Speech issue, I can look into it. if you are faster at it, I would suggest adding a console log here in the stateReducer and check the `type`, `changes` and `state`. If there is a state change in the select, then it should pass here.

Reply:
Great idea!